### ISSUE TITLE
When will you making italic for this font?????? 

### ISSUE THREAD

**Issue by KunalTanwar** [tonsky/FiraCode#1266]
Please add italic font also. Whenever I turned on italics in vscode it just make regular slanted which kinda look ugly. REQUEST!!

**Comment by wopian** [comment]
https://github.com/tonsky/FiraCode/issues/134#issuecomment-184108982

**Comment by davidvmckay** [comment]
According to #134, @tonsky was waiting for upstream Mozilla Fira to finish support for italics...

but as of 2021 Sep 21, [Mozilla Fira](https://github.com/mozilla/Fira/commits/master) hasn't had a commit in 3 years, and barely any substantial changes in 6 years.

FiraCode is far more active than Fira.

Maybe @tonsky or one of us should fork Fira and rebase FiraCode on that fork?  Then the FiraCode community can govern it's own improvements to core Fira for use in FiraCode.  Or, possibly, the source from Fira:HEAD should be inlined into FiraCode and drop the upstream dependency.

**Comment by tonsky** [comment]
Yes, Fira is not developing. The only solution is to draw all the italics myself, which is a huge task and requires skill I don’t really have. I can’t promise it’s coming anywhere in the near future

**Comment by KunalTanwar** [comment]
maybe we can steal italics from Operator Mono 😂 lmao

**Comment by lsloan** [comment]
If you do add italics, please note my comment in #1306 about different glyphs for some characters, like "a".

**Comment by g-e-o-m-e-t-r-i-c** [comment]
There is an italic version for Mono here: https://github.com/zwaldowski/Fira/tree/zwaldowski/mod-new/otf
Maybe we could patch the ligatures based on that version?
